components/HeadContent.jsx

```diff
@@ -47,6 +48,7 @@ export const HeadContent = ({ title }) => (
     <meta name="hostname" content="burgershot.gg" />
     <meta name="expected-hostname" content="burgershot.gg" />
     <meta name="google-site-verification" content="-1qA4kEDqC2i3_jqGD-UYCCqH7jDWZdgEejT7-4QqkE" />
+    <LanguageSelect {...rest} />
```

Comment:
Shouldn't this be inside `<body>` instead of `<head>`?

Or, this `<Head>` element can be placed inside a div because next.js will hoist it up to the top `<html>` level anyway leaving other elements alone.

Reply:
```jsx
<div>
  <Head>
    <meta tags... />
  </Head>

  <LanguageSelect {...rest} />
</div>
```